database/migrations/2025_04_16_000050_update_jobs_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('results', function (Blueprint $table) {
+            $table->unsignedBigInteger('downloaded_bytes')->nullable();
+            $table->unsignedBigInteger('uploaded_bytes')->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::table('results', function (Blueprint $table) {
```

Comment:
You can remove the `down()` method functions as we don't go back in migrations.

Example: https://github.com/alexjustesen/speedtest-tracker/blob/main/database/migrations/2024_11_23_021744_add_healthy_to_results_table.php